README.md

```diff
@@ -11,34 +11,42 @@ Tekton !
 
 Download the latest binary executable for your operating system:
 
-* [Max OS X](https://github.com/tektoncd/cli/releases/download/v0.1.0/tkn_0.1.0_Darwin_x86_64.tar.gz)
+* [Max OS X](https://github.com/tektoncd/cli/releases/download/v0.1.2/tkn_0.1.2_Darwin_x86_64.tar.gz)
```

Comment:
we should really just do latest no ?

Reply:
we could change that once again, but I don't do a new release just for it :joy: